tests/test_meshtastic_utils_connect_paths.py

```
@@ -357,6 +357,57 @@ def test_connect_meshtastic_logs_firmware_version_on_success(
+        "user": {"shortName": "Node", "hwModel": "HW"}
+    }
+    startup_packet = {
+        "from": 123,
+        "to": 456,
+        "decoded": {"text": "startup packet", "portnum": "TEXT_MESSAGE_APP"},
+        "channel": 0,
+        "id": 999,
+        "rxTime": 94_900.0,
+    }
+
+    def _subscribe_side_effect(callback, topic):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_subscribe_side_effect`

Add return type annotation: `None`

(ANN202)